packages/trader/src/App/Components/Elements/market-is-closed-overlay.jsx

```diff
@@ -1,22 +1,60 @@
+import classNames from 'classnames';
 import React from 'react';
-import { Button } from '@deriv/components';
+import PropTypes from 'prop-types';
+import { Button, Text } from '@deriv/components';
 import { localize, Localize } from '@deriv/translations';
+import MarketCountdownTimer from './market-countdown-timer.jsx';
 
-const MarketIsClosedOverlay = ({ onClick }) => (
-    <div className='market-is-closed-overlay'>
-        <p>
-            <Localize i18n_default_text='Market is closed.' />
-        </p>
-        <p>
-            <Localize i18n_default_text='Try Synthetic Indices which simulate real-world market volatility and are open 24/7.' />
-        </p>
-        <Button
-            className='market-is-closed-overlay__button'
-            onClick={onClick}
-            text={localize('Try Synthetic Indices')}
-            primary
-        />
-    </div>
-);
+const MarketIsClosedOverlay = ({ is_eu, is_market_available, is_market_close_overlay_loading, onClick }) => {
+    let message = null;
+    let button = null;
+    if (is_eu) {
+        button = (
+            <Button
+                className='market-is-closed-overlay__button'
+                onClick={onClick}
+                text={localize('See open markets')}
+                primary
+            />
+        );
+    } else {
+        button = (
+            <Button
+                className='market-is-closed-overlay__button'
+                onClick={onClick}
+                text={localize('Try Synthetic Indices')}
+                primary
+            />
+        );
```

Comment:
- The only difference between the `button`s is the `text` prop, you can save some lines of code and just render this inline by using a ternary.
- You can conditionally render the `message` too, e.g.:

```JS
{!is_eu && (
    <Text align='center' as='p' className='market-is-closed-overlay__main-message' size='xs'>
        <Localize i18n_default_text='In the meantime, try our synthetic indices. They simulate real-market volatility and are open 24/7.' />
    </Text>
)}
```

Reply:
Improved a lot more 👍🏻